### PR TITLE
[zh]translate Container Runtime Interface(CRI)

### PR DESCRIPTION
**Pull request by howieyuen**

### CODE REVIEW

File: content/zh/docs/concepts/architecture/cri.md [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+{{<glossary_tooltip text="container runtime" term_id="container-runtime">}} on
+each Node in your cluster, so that the
+{{< glossary_tooltip text="kubelet" term_id="kubelet" >}} can launch
+{{< glossary_tooltip text="Pods" term_id="pod" >}} and their containers.
+
+{{< glossary_definition prepend="The Container Runtime Interface (CRI) is" term_id="container-runtime-interface" length="all" >}}
+-->
+CRI 是一个插件接口，它使 kubelet 能够使用各种容器运行时，无需重新编译集群组件。
+
+你需要在集群中的每个节点上都有一个正在工作的
```
Comment (tengqm) [inline content/zh/docs/concepts/architecture/cri.md:29]: working 在这个上下文意味着“可以正常工作的”，而不是“正在工作的”

File: content/zh/docs/concepts/architecture/cri.md [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+{{<glossary_tooltip text="容器运行时" term_id="container-runtime">}}，
+这样
+{{< glossary_tooltip text="kubelet" term_id="kubelet" >}} 能启动
+{{< glossary_tooltip text="Pod" term_id="pod" >}} 及其容器。
+
+{{< glossary_definition prepend="容器运行时接口（CRI）是" term_id="container-runtime-interface" length="all" >}}
```
Comment (tengqm) [inline content/zh/docs/concepts/architecture/cri.md:35]: 👍

File: content/zh/docs/concepts/architecture/cri.md [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+<!--
+The kubelet acts as a client when connecting to the container runtime via gRPC.
+The runtime and image service endpoints have to be available in the container
+runtime, which can be configured separately within the kubelet by using the
+`--image-service-endpoint` and `--container-runtime-endpoint` [command line
+flags](/docs/reference/command-line-tools-reference/kubelet)
+-->
+当通过 gRPC 连接到容器运行时时，kubelet 充当客户端。
+运行时和镜像服务端点必须在容器运行时中可用，可以使用
+[命令行标志](/docs/reference/command-line-tools-reference/kubelet)的
```
Comment (tengqm) [inline content/zh/docs/concepts/architecture/cri.md:52]: 这个我们翻译了的

```suggestion
[命令行标志](/zh/docs/reference/command-line-tools-reference/kubelet)的
```

File: content/zh/docs/concepts/architecture/cri.md [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+For Kubernetes v{{< skew currentVersion >}}, the kubelet prefers to use CRI `v1`.
+If a container runtime does not support `v1` of the CRI, then the kubelet tries to
+negotiate any older supported version.
+The v{{< skew currentVersion >}} kubelet can also negotiate CRI `v1alpha2`, but
+this version is considered as deprecated.
+If the kubelet cannot negotiate a supported CRI version, the kubelet gives up
+and doesn't register as a node.
+-->
+对 Kubernetes v{{< skew currentVersion >}}，kubelet 偏向于使用 CRI `v1` 版本。
+如果容器运行时不支持 CRI 的 `v1` 版本，那么 kubelet 会尝试其他协商任何旧的支持版本。
```
Comment (tengqm) [inline content/zh/docs/concepts/architecture/cri.md:66]:
```suggestion
如果容器运行时不支持 CRI 的 `v1` 版本，那么 kubelet 会尝试协商任何旧的其他支持版本。
```

File: content/zh/docs/concepts/architecture/cri.md (hunk 2 above)
Comment (howieyuen) [reply content/zh/docs/concepts/architecture/cri.md:32]: 这里，在注释中引用`container-runtime-interface` 会导致渲染结果成这样：
截屏2022-02-23 15 56 28

我尝试了单独注释，删除空行都不行，最后只能删除，知道这是为什么吗？

Comment (tengqm) [reply content/zh/docs/concepts/architecture/cri.md:32]: 这些 shortcode 都被 abuse 了 。。。

**Comment by jihoon-seo** [comment]
This PR is adding [`.hugo_build.lock`](https://github.com/kubernetes/website/pull/31859/files#diff-4b60d2826085e792088186acd4de31d5f4639e950bc66c688f9b36eb38fe79ed) file.
I think that it would be better to exclude this change from this PR.
How about you folks? 😊

---

Preview: https://deploy-preview-31859--kubernetes-io-main-staging.netlify.app/zh/docs/concepts/architecture/cri/

**Comment by howieyuen** [comment]
> This PR is adding [`.hugo_build.lock`](https://github.com/kubernetes/website/pull/31859/files#diff-4b60d2826085e792088186acd4de31d5f4639e950bc66c688f9b36eb38fe79ed) file. I think that it would be better to exclude this change from this PR. How about you folks? 😊
> 
> Preview: https://deploy-preview-31859--kubernetes-io-main-staging.netlify.app/zh/docs/concepts/architecture/cri/

oops...my bad

**Comment by howieyuen** [comment]
cc @mengjiao-liu

**Comment by mengjiao-liu** [comment]
/lgtm 
Thanks!

**Pull request merged**